app/assets/javascripts/strings.js

```diff
@@ -49,3 +49,5 @@ export function StringImportError({errorCount}) {
 
 /** @password_change */
 export const STRING_FAILED_PASSWORD_CHANGE = "There was an error re-encrypting your items. Your password was changed, but not all your items were properly re-encrypted and synced. You should try syncing again. If all else fails, you should restore your notes from backup.";
+
+export const STRING_IMPORT_FAILED_NEWER_BACKUP = "This backup file was created using a newer version of the application and cannot be imported here. Please update your application and try again.";
```

Comment:
Why not match this with `develop` value to make merging easier?

Reply:
I agree in principle but merging won't be feasible because we're going from a .js to .ts file (and with significant changes), so every change I make in `master` I backport immediately to `develop`. I don't think it's much of an issue here.

Reply:
I betcha when we go to merge whenever from now you're going to wonder, why on earth did I set these string values different if I created them at the same time 🤔